gateway-ha/src/test/java/com/lyft/data/gateway/ha/router/TestSpecificDbResourceGroupsManager.java

```diff
@@ -0,0 +1,235 @@
+package com.lyft.data.gateway.ha.router;
```

Comment:
This just seems to be a duplicate of TestResourceGroupManager. Can we just Extend that base class & just override the setup where the schemaName is set. 

In the base class default schemaName=null unless overriden

Reply:
do you mean extend the TestResourceGroupManager and override the setup, wont that still be a duplication of test methods then in the derived class?

Reply:
synced offline, updated with latest changes..